contracts/IdentityOracle.sol

```diff
@@ -244,27 +192,7 @@ contract IdentityOracle is DAOUpgradeableContract, ChainlinkClient, Ownable {
     function getChainlinkToken() public view returns (address) {
```

Comment:
isnt chainlinkTokenAddress alraedy public? where is defined?

Reply:
fixed by https://github.com/GoodDollar/ChainlinkIdentityOracle/pull/5/commits/f05ce99726a869f1dfaf944456647138eaf25fa3